src/components/AppComponents/TimeoutPopup/index.tsx

```diff
@@ -48,12 +48,12 @@ export default function TimeoutPopup(props) {
   useEffect(() => {
     if (!show) {
       // Reset countdown and related states if show is false
-      dispatch({ type: 'UPDATE_TIME_REMAINING', payload: 60 });
+      dispatch({ type: 'UPDATE_TIME_REMAINING', payload: 120 });
       dispatch({ type: 'UPDATE_SCREEN_READER_COUNTDOWN', payload: '' });
```

Comment:
Changing these will effect all journeys and the specific requirements are that we only start the visual countdown at 1 minute, and also that we update the screen readers also at that point. So if we need to change this it should be an enhancement not a bug.

Reply:
Ah right, so it is meant to show at 1 minute? 

The ticket mentions that it should show at 2 minutes.

Reply:
Let us know if you need any information or demo, happy to help :)